vendor/github.com/kubernetes-incubator/external-storage/snapshot/pkg/client/client.go

```diff
@@ -183,7 +204,27 @@ func CreateCRDV1(clientset apiextensionsclient.Interface) error {
 		glog.Fatalf("failed to create VolumeSnapshotResource: %#v, err: %#v",
 			res, err)
 	}
-	return nil
+	return wait.PollImmediate(retryInterval, timeout, func() (bool, error) {
```

Comment:
Duplicate code, use a common function with above

Reply:
@ram-infrac I am assuming this is just vendor'ed library code which you didn't have to add or is this something that you added?

Reply:
v1/v1beta1 crds both have separate api client,  it can't be avoided. Earlier we used to  list all `volumesnapshotdata` to check if crds are registered in `WaitForSnapshotResource`